content/it/function-as-a-service.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Function as a Service (FaaS)
+status: Completed
+category: Tecnologia
+tags: ["infrastruttura", "", ""]
+---
+
+## Cos'è
+
+Function as a Service (FaaS) è un tipo di [servizio](/it/service/) di [cloud computing](/it/cloud-computing/) [serverless](/it/serverless/) 
+che consente l'esecuzione di codice in risposta agli eventi senza mantenere la complessa infrastruttura
+tipicamente associata alla creazione e all'avvio di applicazioni a [microservizi](/it/microservices/).
+Con FaaS, gli utenti gestiscono solo funzioni e dati mentre il provider cloud gestisce l'applicazione.
+Ciò consente agli sviluppatori di ottenere le funzioni di cui hanno bisogno senza pagare per i servizi quando il codice non è in esecuzione.
+Alcuni esempi FaaS popolari includono: [AWS Lambda di Amazon](https://aws.amazon.com/it/lambda/),
+[Google Cloud Functions](https://cloud.google.com/functions/) e [Microsoft Azure Functions](https://azure.microsoft.com/it-it/products/functions/).
+ 
+
+## Quale problema affronta
+
+In uno scenario tradizionale on-premise, un'azienda gestisce e mantiene il proprio data center.
+L'azienda deve investire in server, storage, software e altre tecnologie
+e potenzialmente assumere uno staff IT o appaltare l'acquisto, la gestione e l'aggiornamento di tutte le apparecchiature e le licenze.
```

Comment:
```suggestion
e potenzialmente assumere uno staff IT o dei consulenti per l'acquisto, la gestione e l'aggiornamento di tutte le apparecchiature e le licenze.
```
con riferimento al fatto che di queste attivita se ne devono occupare o lo staff IT o i consulenti.